core/frontend/src/tile/map/MapTileTree.ts

```diff
@@ -1001,9 +1001,11 @@ export class MapTileTreeReference extends TileTreeReference {
 
     for (; treeIndex < this._layerTrees.length; treeIndex++) {
       const layerTreeRef = this._layerTrees[treeIndex];
+      const isImageryMapLayer = layerTreeRef instanceof ImageryMapLayerTreeReference;
       // Load tile tree for each configured layer.
-      // Note: Non-visible layer are also added to allow proper tile tree scale range visibility reporting.
+      // Note: Non-visible imagery layer are always added to allow proper tile tree scale range visibility reporting.
       if (layerTreeRef && TileTreeLoadStatus.NotFound !== layerTreeRef.treeOwner.loadStatus

```

Comment:
Suggestion: This conditional is getting a bit unwieldy. Maybe extract it to a function or extract each of the sub-conditions into its own variable?

Reply:
should be better now

Reply:
@ben-polinsky